### ISSUE TITLE
Platform Monitoring: Implement endpoint monitoring

### ISSUE THREAD

**Issue by akachru-github** [icgc-argo/roadmap#361]
Implement monitoring based off of the Platform monitoring definition in the wiki (all except for Vault, which is a special scenario as it depends on the implementation of the private/public ingress):
https://wiki.oicr.on.ca/pages/viewpage.action?spaceKey=icgcargotech&title=Monitoring

**Comment by jmimico** [comment]
![Screen Shot 2020-04-09 at 10.04.58 AM.png](https://images.zenhubusercontent.com/5daf4d79879c55000190c9ba/73eedd96-d701-435d-a98c-cfd8107b7503)

**Comment by jmimico** [comment]
As per the wiki page, I've implemented some basic web monitors. Vault and lectern are not responding yet. Not sure who can help resolve those remaining two monitors @akachru-github

**Comment by jmimico** [comment]
I've configured these alerts to go to #argo-platform-alerts in the OICR slack workspace.

**Comment by akachru-github** [comment]
@blabadi and @joneubank any thoughts Vault and Lectern being unreachable?

**Comment by blabadi** [comment]
@henro001 is the network guy, we don't have any documentation of the current network setup to refer to

**Comment by henro001** [comment]
lectern is good, vault will be fixed once we have dedicated private ingress. vault ingress has been deployed with private ingress IPs. Once there is private ingress, DNS will need to updated

**Comment by akachru-github** [comment]
Will break off vault into it's own ticket and make it blocked by the ingress task. 
New ticket: #380

**Comment by akachru-github** [comment]
This is now complete with alerts being piped to #argo-platform-alerts in the OICR slack workspace. Vault remains, but is captured in #380.

Closing.
